content/ew-issue-z200-[2017-03-29].yaml

```diff
@@ -101,30 +125,68 @@ content:
                 description: ""
                 link: https://www.youtube.com/watch?v=ZCHFjGSCqP4
             -
+                title: "Ember.js' future, with Tom Dale"
+                description: ""
+                link: https://www.linkedin.com/pulse/emberjs-future-tom-dale-greg-leffler
+            -
                 title: "Creating an Ember Addon - Scott Batson"
                 description: ""
                 link: https://www.youtube.com/watch?v=bhJ6YzBIoWo
+            -
+                title: "Audacious Enterprise GIS Apps with Ember.js"
+                description: ""
+                link: https://www.youtube.com/watch?v=7Bvr7cEQNCg
+    
+    -
+        heading: "So Ember 2017 Videos"
+        headlines:
+            -
+               title: "Keynote - Luke Melia"
+               description: ""
+               link: https://www.youtube.com/watch?v=FJp2f0VHSzU
+            -
+               title: "A journey through Ember.js glue - Mike North"
+               description: ""
+               link: https://www.youtube.com/watch?v=Femu3l8eS7U
+            -
+               title: "DOMless* Rendering with Ember - Miguel Andrade"
+               description: ""
+               link: https://www.youtube.com/watch?v=nkiZ_cWoQMk
+            -
+               title: "Failing Fast: Haskell Inspired Error Handling in Ember - Daniel Chappel"
+               description: ""
+               link: https://www.youtube.com/watch?v=BssIi9TDYFI
+            -
+               title: "Lazy Loading Engines Anything but Lazy - Trent Willis"
+               description: ""
+               link: https://www.youtube.com/watch?v=7ZP9-AIvrBU
+            -
+               title: "Building Accessible Applications: A Reasonable Approach - Melanie Sumner"
+               description: ""
+               link: https://www.youtube.com/watch?v=qxvNXBT03AY
 
 
     -
         heading: "Events"
         headlines:
             - 
-                title: Columbus - 4th April
-                link: "https://www.meetup.com/ember-columbus/events/238743363/"
-            - 
-                title: Seattle - 7th April
+                title: Seattle - 6th April
                 link: "https://www.meetup.com/Ember-js-Seattle-Meetup/events/238136145/"
             - 
-                title: Ember.JS DC - 7th April
+                title: Washington DC - 6th April
```

Comment:
Also, forgot to mention that sometimes the API doesn't return a nice city so the meetup name is used. I usually manually change to a more recognisable city name.